Sources/Datadog/Core/AutoInstrumentation/URLSessionSwizzler.swift

```diff
@@ -34,8 +34,9 @@ internal class URLSessionSwizzler {
     }
 
     static var hasSwizzledBefore = false
+
     @discardableResult
-    func swizzleOnce(using interceptor: @escaping RequestInterceptor) -> Bool {
+    func swizzleOnce(using interceptor: @escaping RequestInterceptor) -> Bool { // swiftlint:disable:this attributes
```

Comment:
this line doesn't give me warning, which version of swiftlint are you using?

Reply:
Yes, it's weird - CI is also good, but if I run the CI locally it fails.
```
❯ swiftlint version
0.38.0
```

<img width="1984" alt="Screenshot 2020-06-16 at 09 40 48" src="https://user-images.githubusercontent.com/2358722/84754095-2c4c4900-afc0-11ea-94e6-3b4c81047d8c.png">

Reply:
looks like you need to update your swiftlint, mine is `0.39.1`

Reply:
👍 works in `0.39.1`. Removed that change.